heudiconv/utils.py

```diff
@@ -319,7 +319,7 @@ def load_heuristic(heuristic):
         path, fname = op.split(heuristic_file)
         try:
             old_syspath = sys.path[:]
-            sys.path.append(path)
+            sys.path[0:0]=[path]
```

Comment:
thanks.  I think (can't come up with a use case when not) it indeed would be better to consider heuristic file path first.
I would though avoid "tricky" `[0:0]` but use explicit `.insert`:
```suggestion
            sys.path[0:0]=[path]
            sys.path.insert(0, path)
```